### ISSUE TITLE
(SuperPutty.install) Should depend on Putty.install

### ISSUE THREAD

**Issue by oninoshiko** [mkevenaar/chocolatey-packages#80]
## Expected Behavior
The putty.install version of putty is installed, or at least used if present.

## Current Behavior
Even with putty.install installed, it installs putty.portable

## Possible Solution
Change dependency to putty.install

## Steps to Reproduce (for bugs)
1. Install putty.install
2. install superputty.install
3. note the installation of putty.portable

## Context
Results in two different versions of putty installed. 
The reason for this is package maintainers for putty _insist_ on the putty meta-package depending on putty.portable rather then doing what _every other chocolatey package does_ and depending on the .install (see: https://github.com/chocolatey-community/chocolatey-coreteampackages/issues/1177)

## Your Environment

* Package Version used: 1.4.0.9
* Operating System and version: Win 10 Pro 20H2 19042.630
* Chocolatey version: 0.10.15
* Install/uninstall gist:

**Comment by mkevenaar** [comment]
Hi @oninoshiko 

Thank you for submitting this issue. First of all, I would like to say that I totally agree with you, however I think I can't change it anymore. Let me try to explain why:

When I took over the maintenance of this package, I inherited the decisions made by the original maintainer.

I would have loved to match the `superputty.install` package with `putty.install`, however since the first submission of this package it has always been dependent on the `putty` package.

If I would change this now, this would result in a braking change for thousands of users.

Back in the day, PuTTY did not have an installer for their software, it always has been portable software. I think that this is why the SuperPuTTY package always depended on the portable version of PuTTY

I am not closing this one (yet), but I need to think this one over a bit more. 

Cheers,

Maurice
